README.md

```diff
@@ -255,6 +255,67 @@ useminPrepare: {
 
 The given steps or post-processors may be specified as strings (for the default steps and post-processors), or as an object (for the user-defined ones).
 
+### warnMissing
+
+Type: 'boolean'
+Default: `nil`
+
+Whether usemin should issue a warning when it can't find one of the sources
+referenced in a block. For backwards compatibility, this is off by default.
+
+### resolveSource
+
+Type: 'function'
+Default: `nil`
```

Comment:
@sindresorhus good catch. 

This is a rebase from the #306 and I'm not the author of this work is @eli-collins, as you can see from my first commit message. 

The author added this comment in his PR:
> I've added a warnMissing option, which causes ConfigWriter to issue a warning/error
for any source references it can't resolve (ala issue #260). This defaults to off,
so as not to break things for people.

IMO it might be better to defaults to `true`, and only users who might need this option would set it to `false`.